torchgeo/models/resnet.py

```diff
@@ -138,6 +139,7 @@ class ResNet18_Weights(WeightsEnum):  # type: ignore[misc]
             'publication': 'https://arxiv.org/abs/2306.09424',
             'repo': 'https://github.com/microsoft/torchgeo',
             'ssl_method': 'moco',
+            'bands': _satlas_landsat_bands,
```

Comment:
`_satlas_landsat_bands` contains 11 bands, but this model was only trained on 7 (see `in_chans`). None of these models are Satlas models, so there's no reason to believe they should have the same bands as Satlas.

Reply:
> `_satlas_landsat_bands` contains 11 bands, but this model was only trained on 7 (see `in_chans`). None of these models are Satlas models, so there's no reason to believe they should have the same bands as Satlas.

@adamjstewart , Hi, sorry for the confusion on my part, now I have made the appropriate changes, can you review the PR, and if there are no more concerns, can you add the "hacktoberfest-accepted" label to the PR, and then merge it before 1 November, I will be very thankful.